modules/sno-du-conn-tuning-the-performance-patch.adoc

```diff
@@ -0,0 +1,43 @@
+
+ // Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-connected.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-conn-tuning-the-performance-patch_{context}"]
+= Extending the tuned profile configuration
+
+Apply this `tuned` CR to extend the Tuned profile with additional configuration necessary to realize the performance targets for DU workloads.
+
+.Procedure
+
+*  Tune the performance patch using the following example:
++
+[source,yaml]
+----
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: performance-patch
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+    - data: |
+        [main]
+        summary=Configuration changes profile inherited from performance created tuned
+        include=openshift-node-performance-openshift-node-performance-profile
+        [bootloader]
+        cmdline_crash=nohz_full=2-19,22-39
+        [sysctl]
+        kernel.timer_migration=1
+        [scheduler]
+        group.ice-ptp=0:f:10:*:ice-ptp.*
+        [service]
+        service.stalld=start,enable
```

Comment:
we now disable chronyd in here: https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.10/ztp/source-crs/TunedPerformancePatch.yaml#L25